src/components/ListItem.jsx

```diff
@@ -1,12 +1,30 @@
 import './ListItem.css';
+import { deleteItem } from '../api/firebase.js';
+import { useRef } from 'react';
 
 export function ListItem({
 	name,
 	isDefaultChecked,
 	itemId,
 	setIsChecked,
 	setCheckedItemId,
+	listToken,
 }) {
+	const dialogRef = useRef();
+
+	function confirmDelete() {
```

Comment:
🔍 This function's name is a little ambiguous. The modal asks our user to confirm their delete action, yes, but `confirmDelete` also kind of reads like "we have confirmed the delete; let's go ahead and do it." I think we can call this `openModal`, because that's what it does.

Reply:
Agree. :-)